goreleaser.hsm.yaml

```diff
@@ -0,0 +1,282 @@
+version: 2
+
+project_name: bao
```

Comment:
This is fine, but we'll have to keep in mind that any update to a section like `docker` or something probably needs to be redone in several places.

Reply:
Yes, this is unfortunately the case.